### PR TITLE
4/13 cake - Chocolatey Cake

### PR DESCRIPTION
**Pull request by glubtier**
Based on the cake from Matilda.

Item texture: Rose
Block textures: Vinelli
Code: glubtier

### CODE REVIEW

**Comment by glubtier** [comment]
Also -- if someone wants to make changes to things like distribution, name, add them to consort loot tables, etc., have at it!

File: src/main/java/com/mraof/minestuck/data/recipe/MinestuckCombinationsProvider.java [hunk 1]
```diff
@@ -1060,6 +1060,7 @@ public static void buildRecipes(Consumer<FinishedRecipe> consumer)
 		CombinationRecipeBuilder.of(MSBlocks.CARROT_CAKE.get()).input(Items.CAKE).or().input(Items.CARROT).build(consumer);
 		CombinationRecipeBuilder.of(MSBlocks.LARGE_CAKE.get()).input(Items.CAKE).or().input(Items.BONE_MEAL).build(consumer);
 		CombinationRecipeBuilder.of(MSBlocks.PINK_FROSTED_TOP_LARGE_CAKE.get()).input(MSItems.LARGE_CAKE.get()).and().input(Items.PINK_DYE).build(consumer);
+		CombinationRecipeBuilder.of(MSBlocks.CHOCOLATEY_CAKE.get()).input(Items.CAKE).or().input(Items.COCOA_BEANS).build(consumer);
```
[the remark this answers is not in the document]
Comment (glubtier) [reply src/main/java/com/mraof/minestuck/data/recipe/MinestuckCombinationsProvider.java:1063]: Whoops yep, wasn't intentional, just forgot. Added.

**Review by kirderf1 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - Cast Iron Frame, Sheet, and Tile
 - Sulfur Pools and Cast Iron features in Heat Lands
 - Wooden Lamp and Heat Lamp blocks, used in Wood and Heat lands respectively
-
+- Chocolatey Cake - the most scrumptious cake in the entire world!
```
Comment (Dweblenod) [inline CHANGELOG.md:38]: add a space between this line and ### Changed

**Review by Dweblenod — APPROVED** [review]
Once the changelog fix is done, this has my approval!

File: CHANGELOG.md (hunk 2 above)
Comment (glubtier) [reply CHANGELOG.md]: Done

**Pull request merged**